onfido-java/src/main/java/com/onfido/managers/DocumentManager.java

```diff
@@ -0,0 +1,36 @@
+package com.onfido.managers;
+
+import com.onfido.api.Config;
+import com.onfido.api.ResourceManager;
+import com.onfido.exceptions.OnfidoException;
+import com.onfido.models.Document;
+import okhttp3.MediaType;
+import okhttp3.MultipartBody;
+import okhttp3.RequestBody;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URLConnection;
+
+public class DocumentManager extends ResourceManager {
+
+    public DocumentManager(Config config) {
+        super("documents/", config);
+    }
+
+    public void uploadDocument(InputStream inputStream, String fileName, Document.Request request) throws IOException, OnfidoException {
```

Comment:
I think it would be more readable to do `onfido.document.upload(...)` rather than `onfido.document.uploadDocument(...)` i.e. I'd rename this to `upload` (and rename the `upload` method in ResouceManager to something else)

Same for `downloadDocument`

Plus that way, once upload/download of photos and videos is added, the method names should be consistent

Reply:
The API also returns the document object so it would be good to parse that into a `com.onfido.models.Document` instance too

Reply:
Done